NetKAN/TweakScaleCompanion-ReStockPlus.netkan

```diff
@@ -0,0 +1,31 @@
+  
+{
+    "spec_version"     : "v1.2",
+    "$kref"            : "#/ckan/spacedock/2424",
+    "$vref"            : "#/ckan/ksp-avc",
+    "identifier"       : "TweakScaleCompanion-ReStockPlus",
+    "name"             : "TweakScale Companion for the ReStockPlus AddOn",
+    "abstract"         : "Adds TweakScale /L patches for ReStockPlus.",
+    "license"          : [ "GPL-2.0", "restricted" ],
```

Comment:
I assume you chose GPL-2.0 and restricted because some of the contained files use one license and some use the other. That is **not** what a license array means:

https://github.com/KSP-CKAN/CKAN/blob/master/Spec.md#license

> The following are both valid, the first describing a mod released under the BSD license, the second under the _user's choice_ of BSD-2-clause or GPL-2.0 licenses.

> If different assets in the mod have different licenses, the _most restrictive_ license should be specified, which may be restricted.

So this metadata as written would mean that users can choose to use everything in the mod as GPL-2.0 or as restricted, at their option.

Reply:
And that's exactly what it means, the Add'On is double licensed - the user can choose the license he wants to adopt.

Reply:
OK, sounds good.

Reply:
The restricted license allows the user to embedded the Add'On on packages as long the material is not changed. It's a safe bet to be used for Authors that embed add'ons and don't use GPL as the license.